records/new/v23.4.0-announcement.json

```
@@ -0,0 +1,5 @@
+{
+  "action": "post",
+  "account": "NODEJS_ORG",
+  "richText": "Thanks to @targos.dev and @aduh95.bsky.social for a new Node.js Current release: [v23.4.0](https://nodejs.org/en/blog/release/v23.4.0)"
```

Hmm, actually I don’t think the markdown syntax works with rich text detection. It should just be
```suggestion
  "richText": "Thanks to @targos.dev and @aduh95.bsky.social for a new Node.js Current release: https://nodejs.org/en/blog/release/v23.4.0"
```